test/sagemaker_tests/pytorch/training/integration/local/test_distributed_training.py

```diff
@@ -60,6 +63,8 @@ def test_dist_operations_path_gpu_nccl(docker_image, sagemaker_local_session, tm
 
 
 @pytest.mark.skip_gpu
+@pytest.mark.processor("cpu")
+@pytest.mark.integration("nccl")
```

Comment:
Looks like this is a multinode test.

pytest.mark.multinode("multinode(2)") ?

Actually, not sure what "train_instance_count=2" means in the context of local tests.